run.sh

```diff
@@ -40,10 +40,9 @@ fi
 
 $SCRIPT_DIR/docker/linux-$NATIVE_ARCH/build.sh
 
+# FUSE required for AppImage
 docker run --rm \
-    --privileged \
-    --device /dev/fuse \
-    --cap-add SYS_ADMIN \
+    --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined \
```

Comment:
Did you verified if it capable to build appimage without "--privileged" flag? ^__^

Reply:
I tested and it works! Thank you!